db/migrate/20180416124217_create_hw_infos.rb

```diff
@@ -0,0 +1,14 @@
+class CreateHwInfos < ActiveRecord::Migration[5.1]
+  def change
+    create_table :hw_infos do |t|
+      t.integer :cpus
+      t.integer :sockets
+      t.string :hypervisor, limit: 255
+      t.string :arch, limit: 255
+      t.integer :system_id
+      t.string :uuid, limit: 255
```

Comment:
There probably should be a unique index on this column.

Note: `glue` has a unique index.

Reply:
from 'glue' first migration
so there are not unique index on UUID but I will probably add it (but allow `null` values). will also copy other indexes
```ruby
add_index 'hw_infos', ['hypervisor'], name: 'index_hw_infos_on_hypervisor', using: :btree
add_index 'hw_infos', ['system_id'], name: 'index_hw_infos_on_system_id', unique: true, using: :btree
add_index 'hw_infos', ['uuid'], name: 'index_hw_infos_on_uuid', using: :btree
```